vignettes/laminr.Rmd

```diff
@@ -115,40 +115,66 @@ This artifact contains an [`AnnData`](https://anndata.readthedocs.io) object.
 If you prefer a path to a local file or folder, call `path <- artifact$cache()`.
 </div>
 
-# Work with the data
+# Work with the dataset
 
 Once you have loaded a dataset you can perform any analysis with it as you would normally.
 Here, marker genes are calculated for each of the provided cell type labels using [**{Seurat}**](https://satijalab.org/seurat/).
 
 ```{r create-seurat}
 # Create a Seurat object
-seurat <- SeuratObject::CreateSeuratObject(
+seurat_obj <- SeuratObject::CreateSeuratObject(
   counts = as(Matrix::t(adata$X), "CsparseMatrix"),
   meta.data = adata$obs
 )
+# add gene metadata
+seurat_obj[["RNA"]] <- AddMetaData(GetAssay(seurat_obj), adata$var)
 # Set cell identities to the provided cell type annotation
-SeuratObject::Idents(seurat) <- "cell_type"
+SeuratObject::Idents(seurat_obj) <- "cell_type"
 # Normalise the data
-seurat <- Seurat::NormalizeData(seurat)
+seurat_obj <- Seurat::NormalizeData(seurat_obj)
 # Test for marker genes (the output is a data.frame)
 markers <- Seurat::FindAllMarkers(
-  seurat,
-  features = SeuratObject::Features(seurat)[1:100] # Only test a few features for speed
+  seurat_obj,
+  features = SeuratObject::Features(seurat_obj)[1:100] # Only test a few features for speed
 )
 # Display the marker genes
 knitr::kable(markers)
 # Plot the marker genes
-Seurat::DotPlot(seurat, features = unique(markers$gene)) +
+Seurat::DotPlot(seurat_obj, features = unique(markers$gene)) +
   ggplot2::theme(axis.text.x = ggplot2::element_text(angle = 90, vjust = 0.5))
 ```
 
+# Slice the tiledbsoma array store
+
+Alternatively to querying individual datasets, you can also slice the `.tiledbsoma` array store, which stores Census, a concatenated version of most datasets in CELLxGENE.
+
+```{r create-seurat}
```

Comment:
There is a duplicate chunk label which is causing the rendering to fail

````suggestion
```{r slice-tiledb}
````